Bio/GenBank/__init__.py

```diff
@@ -45,7 +42,13 @@
 
 from Bio import BiopythonParserWarning
 from Bio.Seq import Seq
-from Bio import SeqFeature
+from Bio.SeqFeature import (
+    SimpleLocation,
+    Location,
+    Reference,
+    SeqFeature,
+    LocationParserError,
+)
```

Comment:
Minor: Style wise although not formalised we've avoid the bracket multiline imports, I would use multiple import lines here - sorted alphabetically.

(There are some issues about automatically sorting/formatting our imports)

Reply:
done